src/lib/zexe_backend/zexe_backend_common/dlog_based_keypair.ml

```diff
@@ -89,6 +90,10 @@ module Make (Inputs : Inputs_intf) = struct
     let urs_info = Set_once.create () in
     let urs = ref None in
     let degree = 1 lsl Pickles_types.Nat.to_int Rounds.n in
+    (* TODO *)
+    let public_inputs = Unsigned.Size_t.of_int 1 in
```

Comment:
Is there reason of setting "public_inputs" and "size" to 1 rather than 0 ?

Reply:
Good catch, thanks! Updated.